src/Utils/MODES.java

```diff
@@ -1,30 +1,32 @@
 package Utils;
 
 public class MODES {
-    public static String SELECT = "Select";
-    public static String ASSOCIATION_LINE = "AssociationLine";
-    public static String GENERALIZATION_LINE = "GeneralizationLine";
-    public static String COMPOSITION_LINE = "CompositionLine";
-    public static String CLASS = "Class";
-    public static String USE_CASE = "Use Case";
+    public static final String SELECT = "Select";
```

Comment:
We have remove `final` in #18. Also please remove https://github.com/1chooo/uml-editor/pull/23/commits/34010279e4568b830ff7d86a2fa66a5a2555b2b4 and https://github.com/1chooo/uml-editor/pull/23/commits/cb70ce0e97a3bf9af2f7a5b20c713d3371f9d132 because they are not related to this PRs. I think the solution of this problem is you have to sync my `main` branch again and rebase to the version https://github.com/1chooo/uml-editor/commit/a01c34171ee90d31c9529f8a8ba093b78e300c64